core/domain/exp_services.py

```diff
@@ -1143,24 +1490,56 @@ def get_next_page_of_all_non_private_commits(
 
 def _exp_rights_to_search_dict(rights):
     # Allow searches like "is:featured".
+    """
+    Args:
+        rights: ActivityRights. Domain object for the rights/publication status
+            of the exploration.
+
+    Returns:
+        dict.
```

Comment:
Add some info about what this dict is used for; what's its context? If I want to add stuff to this dict in the future, for example, what do I need to bear in mind?